wp-content/themes/twentyfifteen/css/editor-style.css

```diff
@@ -475,3 +475,7 @@ body.rtl {
 .rtl table caption {
 	text-align: right;
 }
+
+.some-class {
+	background-color: #900;
```

Comment:
we do not allow some-class to have that background color some-class is always #009